unittest/CMakeLists.txt

```
@@ -43,7 +43,8 @@ set(${PROJECT_NAME}_CPP_TESTS
     test_friction_cone
     test_wrench_cone
     test_boxqp
-    test_solvers)
+    test_solvers
+    test_activation_bounds)
```

Let's move this new test to `test_activate.cpp`.
```suggestion
    test_solvers)
```